packages/auth/e2e/user.e2e.js

```diff
@@ -396,4 +396,20 @@ describe('auth().currentUser', () => {
       await firebase.auth().signOut();
     });
   });
+
+  describe('user.metadata', () => {
+    it("should have the properties 'lastSignInTime' & 'creationTime' which are ISO strings", async () => {
+      const random = Utils.randString(12, '#aA');
+      const email = `${random}@${random}.com`;
+
+      const { user } = await firebase.auth().createUserWithEmailAndPassword(email, random);
+
+      const { metadata } = user;
+
+      should(metadata.lastSignInTime).be.a.String();
+      should(metadata.creationTime).be.a.String();
```

Comment:
Could we do an additional check here to make sure the date is valid, e.g. new date the ISO date string and check the year is the same as the current dates year - this check ensure then that the number that comes from native also valid (seconds vs milliseconds)

```js
(new Date(metadata.lastSignInTime)).getFullYear().should.equal((new Date()).getFullYear());
(new Date(metadata. creationTime)).getFullYear().should.equal((new Date()).getFullYear());
```